### PR TITLE
karate 'Viper Lite' race tune

### PR DESCRIPTION
**Pull request by sugaarK**
This racing tune was developed from the Karate Race tune to suit my new Viper lite ultralite 5" racer project. [Viper Lite](https://rotorbuilds.com/build/29775)

ViperLite

### CODE REVIEW

File: presets/4.3/tune/karate_race_lite.txt [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: karate, race, 5 inch, 5", sugarK, limon, ctzsnooze, KarateBrot, ultralite, viper, viper lite
+#$ AUTHOR: sugarK
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION:
+#$ DESCRIPTION: <img src="https://user-images.githubusercontent.com/83344205/176923532-e7791ff9-aa4f-457e-8222-78bd589c0b5d.png" width="400px" style="margin-left: auto; margin-right: auto; display: block;"/>
```
Comment (KarateBrot) [inline presets/4.3/tune/karate_race_lite.txt:11]:
```suggestion
#$ DESCRIPTION: <img src="https://user-images.githubusercontent.com/19867640/176932353-fdc42c5e-33ae-42d7-b34c-e981b6cffa3f.png" style="max-width: 100%; margin-left: auto; margin-right: auto; display: block;"/>
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]

**Comment by SupaflyFPV** [comment]
Sick Image

**Comment by limonspb** [comment]
plz don't make me comment "plz squash" and always keep 1 commit :)

**Comment by sugaarK** [comment]
> plz don't make me comment "plz squash" and always keep 1 commit in the PR ready to merge :)

I would have but I was at work and can't nerd to the required level on my fone

File: presets/4.3/tune/karate_race_lite.txt (hunk 1 above)
Comment (sugaarK) [reply presets/4.3/tune/karate_race_lite.txt]: done

**Review by KarateBrot — APPROVED** [review]

File: presets/4.3/tune/karate_race_lite.txt [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+#$ AUTHOR: sugarK
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION:
+#$ DESCRIPTION: <img src="https://user-images.githubusercontent.com/19867640/176932353-fdc42c5e-33ae-42d7-b34c-e981b6cffa3f.png" style="max-width: 100%; margin-left: auto; margin-right: auto; display: block;"/>
+#$ DESCRIPTION: <h1>Karate Viper Lite race</h1>
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: This racing tune was developed from the Karate Race tune ti suit my new Viper lite ultralite 5" racer project. [Viper Lite](https://rotorbuilds.com/build/29775)
```
Comment (limonspb) [inline presets/4.3/tune/karate_race_lite.txt:15]: **to** suit :)

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**